src/backend/opencl/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 FIND_PACKAGE(OpenCL REQUIRED)
 
+OPTION(USE_CLBLAST "Build ArrayFire with the CLBlast BLAS library for the OpenCL backend" OFF)
+OPTION(USE_CLBLAS "Build ArrayFire with the clBLAS BLAS library for the OpenCL backend" ON)
```

Comment:
OK, so it will be an either CLBlast/clBLAS option such that we don't have verify whether only one of them is set? Makes sense. Do you perhaps have an example of this construct in the library already? Otherwise I'll figure it out myself based on the link.